src/main/java/chess/domain/piece/Pawn.java

```diff
@@ -0,0 +1,67 @@
+package chess.domain.piece;
+
+import chess.domain.point.Point;
+import chess.domain.point.Rank;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public final class Pawn extends Piece {
+
+    private static final int TWO_RANK = 2;
+    private static final Map<Team, Pawn> POOL = Map.of(
+            Team.WHITE, new Pawn(Team.WHITE),
+            Team.BLACK, new Pawn(Team.BLACK)
+    );
+
+    private Pawn(Team team) {
+        super(team);
+    }
+
+    static Pawn from(Team team) {
+        return POOL.get(team);
+    }
+
+    @Override
+    public boolean isMovable(Point currentPoint, Point nextPoint, Piece target) {
+        Team team = getTeam();
+        List<Point> movablePoints = findMovablePoints(currentPoint, team);
+        return movablePoints.contains(nextPoint) &&
+                (movableDiagonal(currentPoint, nextPoint, target) || movableStraight(currentPoint, nextPoint, target));
+
+    }
+
+    private List<Point> findMovablePoints(Point currentPoint, Team team) {
+        List<Point> points = new ArrayList<>();
+        int forwardDirection = team.forwardDirection();
+
+        if (isStartPosition(currentPoint)) {
+            findMovablePoint(points, currentPoint,
+                    Direction.of(0, TWO_RANK * forwardDirection));
+        }
+        for (Direction direction : Direction.findPawnDirections()) {
+            findMovablePoint(points, currentPoint,
+                    Direction.of(direction.file(), direction.rank() * forwardDirection));
```

Comment:
이렇게 자동 개행에 의한 개행이 가독성이 떨어지는 경우, 스스로 개행해주면 더 좋습니다
그리고 direction 관련 코드는 위와 동일한 리뷰입니다. 
```suggestion
            findMovablePoint(
               points,
               currentPoint,
               forwardDirection ? direction : direction.reverse() // 애초에 findWhitePawnDirections / findBlackPawnDirections로 갈라 놓는 것도 방법일듯
           );
```

Reply:
위의 답변과 같이 전체적으로 리팩토링 해보겠습니다.